packages/utilities/psammead-assets/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 2.0.0   | [PR#1646](https://github.com/bbc/psammead/pull/1646) Add SVGs for these services mundo, punjabi. Update SVG name from turkish to turkce.  |
```

Comment:
This is a major version bump since an export had to be renamed from `turkish` to `turkce`, which is the service name. `https://www.bbc.com/turkce`